python-avd/pyavd/_eos_designs/structured_config/core_interfaces_and_l3_edge/utils.py

```diff
@@ -143,8 +143,7 @@ def _get_p2p_data(self: AvdStructuredConfigCoreInterfacesAndL3EdgeProtocol, p2p_
         ):
             node_data = p2p_link.port_channel.nodes_child_interfaces[self.shared_utils.hostname]
             # Port-channel
-            default_channel_id = int("".join(re.findall(r"\d", node_data.interfaces[0])))
-            portchannel_id = node_data.channel_id or default_channel_id
+            portchannel_id = self._get_channel_id(p2p_link, node_data)
```

Comment:
should we try to detect potential duplicates here? (happy for this to be another PR)

with channel id / using id + offset and first_port it feels quite easy to clash in difficult to catch ways

Reply:
We would catch it in the `port_channel_interfaces.append()` call.

Reply:
duh! brain need to start working again :)